content/documentation/staging/features/index.adoc

```diff
@@ -392,84 +392,241 @@ Check the complete link:/documentation/staging/validations/[list of validations]
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/wizard-advanced-options-traffic-policy-v1.24.0.png" title="Traffic Policy">
+   <img src="/images/documentation/features/wizard-advanced-options-traffic-policy-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+=== Worksload Actions
+
+==== Automatic Sidecar Injection
+
+A *Workload* can be individually annotated to control the Sidecar Injection.
+
+A default scenario is to indicate this at *Namespace* level but there can be cases where a *Workload* shouldn't be part of the Mesh or viceversa.
+
+Kiali allows to annotate the Deployment template and propagate this configuration into the Pods.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/workload-actions-sidecar-injection-v1.24.0.png" title="Automatic Sidecar Injection">
+   <img src="/images/documentation/features/workload-actions-sidecar-injection-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+=== Namespace Actions
+
+Kiali offers several actions for a *Namespace* located in the Overview page in any of its views: Expanded, Compacted or Table.
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/overview-table-actions-v1.24.0.png" title="Namespace Actions">
+   <img src="/images/documentation/features/overview-table-actions-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+==== Show
+
+Show actions allows to navigate from a *Namespace* to its specific Graph, Applications, Workload, Services or Istio Config pages.
+
+==== Automatic Sidecar Injection
+
+When https://istio.io/latest/docs/setup/additional-setup/sidecar-injection[Automatic Sidecar Injection] is enabled in the cluster, a *Namespace* can be labeled to enable/disable the injection webhook in the namespace, then new deployments will automatically have a sidecar added to them.
+
+==== Security: Traffic Policies
+
+Kiali can generate Traffic Policies based on traffic for a namespace.
+
+For example, at some point a namespace presents a traffic graph like this:
+
+++++
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/overview-actions-trafficpolicies-graph-v1.24.0.png" title="Traffic Policies: Graph">
+   <img src="/images/documentation/features/overview-actions-trafficpolicies-graph-v1.24.0.png" style="width: 1333px;display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
+++++
+
+And a user may want to add Traffic Policies to secure that communication, in other words, to not allow other traffid different to the represented by the Services and Workloads in the Graph.
```

Comment:
communication, in... --> communication.  In other words, to prevent traffic other than that currently reflected in the Graph's Services and Workloads.